src/lib/css-parse.html

```diff
@@ -89,6 +90,18 @@
       return node;
     },
 
+    // conversion of sort unicode escapes with spaces like `\33 ` (and longer) into
+    // expanded form that doesn't require trailing space `\000033`
+    _expandUnicodeEscapes : function(s) {
+      return s.replace(/\\([0-9a-f]{1,6})\s/gi, function() {

```

Comment:
Actually yes: http://www.w3.org/TR/CSS2/syndata.html#escaped-characters
Together with #2671 it should be solid fix for mentioned issue.